training.py

```diff
@@ -17,16 +17,29 @@
 
 if __name__ == "__main__":
 
+    ########--- Parse for parameters ---########
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--url", dest="url", default=None, type=str, help="setup URL")
+    parser.add_argument("--train_fs", dest="train_fs", required=True, type=str, help="featureset")
+    parser.add_argument("--test_fs", dest="test_fs", required=True, type=str, help="featureset")
+
+    global FLAGS
+    FLAGS, unparsed = parser.parse_known_args()
+    dkubeURL = FLAGS.url
+    train_fs = FLAGS.train_fs
+    test_fs = FLAGS.test_fs
+
     ########--- Read features from input FeatureSet ---########
 
     # Featureset API
-    featureset = DkubeFeatureSet()
-    # Specify featureset path
-    featureset.update_features_path(path=inp_path)
+    authToken = os.getenv("DKUBE_USER_ACCESS_TOKEN")
+    # Get client handle
+    api = DkubeApi(URL=dkubeURL, token=authToken)
 
     # Read features
-    data = featureset.read()  # output: response json with data
-    feature_df = data["data"]
+    feature_df = api.read_featureset(name = train_fs)  # output: data
+
```

Comment:
You can also avoid API token and read directly using DkubeFeatureSet.read_features by specifying path. You can add a comment to this affect. No need to change code.